client/src/pages/admin/Index.js

```diff
@@ -30,8 +34,26 @@ const GQL_SAVE_ADMIN_SETTINGS = gql`
     saveAdminSettings(settings: $settings)
   }
 `
-
+const CLEAR_CACHE = gql`
+  query {
+    clearCache
+  }
+`
+const RELOAD_DICTIONARY = gql`
+  query {
+    reloadDictionary
```

Comment:
shouldn't this be a mutation instead of a query? Maybe also `clearCache`

Reply:
Updated as mutation both clearCache and reloadDictionary